### PR TITLE
Remove Crayons per issue 3741, use click echo functions instead

### PR DESCRIPTION
**Pull request by otherjake**
Thank you for contributing to Pipenv!

### The issue

Remove crayons per issue 3741
Fixes https://github.com/pypa/pipenv/issues/3741

### The fix

Use click.echo, click.secho, click.style instead

### The checklist

* [ ] support colorblind envV with click.style if possible

### CODE REVIEW

**Comment by matteius** [comment]
I think in the long run we need to deprecate the colorblind environment variable and replace it with a more appropriately named variable.  However we do need to consider that we likely had already broken that variable with the current usages of click styling.  @oz123 can you think of a way to support disabling the click styling at a global level in pipenv based on enabling a specific flag?

**Comment by oz123** [comment]
> I think in the long run we need to deprecate the colorblind environment variable and replace it with a more appropriately named variable. However we do need to consider that we likely had already broken that variable with the current usages of click styling. @oz123 can you think of a way to support disabling the click styling at a global level in pipenv based on enabling a specific flag?

The appropriate name would be NO_COLOR, see here: https://no-color.org/

rich, which is already used by pip supports it. In the long run, I would like us to use the same infrastructure as pip, hence I would vote in favor using coloring, and progress bars from rich. It's more work, but it is more sustainable for us.

As for globally disabling click colors as long we  use it, we can steal some code from the [click-extras ](https://github.com/kdeldycke/click-extra/blob/9a5a7e3570eb91a0edb419e1efb15fb9549a1ece/click_extra/colorize.py#L163) project and monkey patch click to disable the color function.

**Review by oz123 — APPROVED** [review]

File: pipenv/core.py [hunk 1]
```diff
@@ -664,10 +671,10 @@ def _cleanup_procs(project, procs, failed_deps_queue, retry=True):
                 project.environment.expand_egg_links()
                 click.echo(
                     "{}".format(
-                        crayons.yellow(
+                        click.style(
                             "Failed initial installation: Failed to overwrite existing "
                             "package, likely due to path aliasing. Expanding and trying "
-                            "again!"
+                            "again!",
```
Comment (matteius) [inline pipenv/core.py:677]: missing `fg="yellow"`

Comment (oz123) [reply pipenv/core.py:677]: Fixed.

File: pipenv/core.py [hunk 2]
```diff
@@ -75,20 +74,22 @@
...
 else:
     INSTALL_LABEL = "   "
     INSTALL_LABEL2 = "   "
     STARTING_LABEL = "   "
 
 # Disable colors, for the color blind and others who do not prefer colors.
-if environments.PIPENV_COLORBLIND:
-    crayons.disable()
+#if environments.PIPENV_COLORBLIND:
+
```
Comment (otherjake) [inline pipenv/core.py:86]: this is the spot where it previously switched the NO_COLOR behavior

Comment (oz123) [reply pipenv/core.py:86]: image

Comment (otherjake) [reply pipenv/core.py:86]: I am implementing a conditional monkeypatch in this spot on style and secho, they are used separately for composing messages that aren't all one color, also tests for each for the old and new variables.  Should an issue be created for deprecating PIPENV_COLORBLIND?

**Pull request merged**